### DISCUSSION TITLE
After upgrade to 2.4.70 the forward sensors servers fail to boot oracle linux

### DISCUSSION THREAD

**Discussion by ghost** [Security-Onion-Solutions/securityonion#13279] [2.4]
### Version

2.4.70

### Installation Method

Security Onion ISO image

### Description

upgrading

### Installation Type

Distributed

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

8

### RAM

32

### Storage for /

800GB

### Storage for /nsm

1TB

### Network Traffic Collection

span port

### Network Traffic Speeds

Less than 1Gbps

### Status

No, one or more services are failed (please provide detail below)

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

The forwarder's server failed to boot oracle linux after rebooting. This happened after upgrading to 2.4.70. I know this mainly belongs in the oracle linux forum. But by any chance any of you encountered it and found the solution. Ours is a DELL server.

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by dougburks** [comment]
I would start with running full hardware diagnostics on your Dell server.
